Tools/FilterTestTool/BiquadFilter.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+
+""" Ardupilot BiquadFilter
```

Comment:
Please change all these to Ardu**P**ilot

Reply:
Thanks Francisco, I forgot some of them